rtos/mbed_lib.json

```diff
@@ -1,6 +1,198 @@
 {
     "name": "rtos",
     "config": {
-        "present": 1
+        "present": 1,
+
+        "taskcnt": {
+            "help": [
+                "Number of concurrent running user threads <1-250>",
+                " Defines max. number of user threads that will run at the same time.",
+                " Default: 6"
```

Comment:
I agree defaults should be defined in the mbed_lib.json for clarity where possibly. Unfortunately there is already quite a bit of logic surrounding these definitions (for example OS_TASKCNT is declared [here](https://github.com/mbedmicro/mbed/blob/master/rtos/rtx/TARGET_CORTEX_M/RTX_Conf_CM.c#L55), and that's only the cortex-M constraints).

I figured it was safer to default defering the declaration to CMSIS.